java/src/jmri/jmrix/loconet/slotmon/SlotMonPane.java

```diff
@@ -159,6 +165,43 @@ void setColumnToHoldButton(JTable slotTable, int column) {
                 .setPreferredWidth(new JButton("  " + slotModel.getValueAt(1, column)).getPreferredSize().width);
     }
 
+    /*
+     * Helper class to format number and optionally make blank when zero
+     */
+    private static class NumberFormatRenderer extends DefaultTableCellRenderer
+    {
+        public NumberFormatRenderer(String pattern, boolean suppressZero) {
+            super();
+            this.pattern = pattern;
+            this.suppressZero = suppressZero;
+            setHorizontalAlignment(JLabel.RIGHT);
+        }
+        @Override
+        public void setValue(Object value)
+        {
```

Comment:
No need to fix, but the code is easier to read if there is an empty line between two methods.